src/05-led-roulette/Cargo.toml

```diff
@@ -1,8 +1,12 @@
 [package]
-authors = ["Jorge Aparicio <jorge@japaric.io>"]
+authors = [
+    "Jorge Aparicio <jorge@japaric.io>",
+    "Christopher J. McClellan <chris.mcclellan203@gmail.com>",
```

Comment:
I understand that you add your name @winksaville, but why this one too?

Reply:
Because Christopher is the author of stm32f-discovery and none of my changes would be possible without his effort. Also, I'm using his [Blinky.rs](https://github.com/rubberduck203/stm32f3-discovery/blob/8214e663ea0e16a7988cf07f40597205273adc95/examples/blinky.rs) code as a basis for `lib.rs`.